src/main/java/hudson/plugins/templateproject/ProxyBuilder.java

```diff
@@ -87,7 +89,16 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher,
 			if (!builder.perform(build, launcher, listener)) {
 				return false;
 			}
-		}
+        String URLConfigureTemplate;
+        if (Jenkins.getInstance().getRootUrl() == null){
+            listener.getLogger().println("Starting template job: " + getProjectName());
+            listener.getLogger().println("Could not get URL from configuration, change URL settings in \"Manage Jenkins->Configure System\"");
+            listener.getLogger().println("If you set this a link to the configuration of the template job will be displayed instead");
+        } else {
+            URLConfigureTemplate = Jenkins.getInstance().getRootUrl() + "job/" + getProjectName() + "/configure";
```

Comment:
That was a requirement from the Release managers who was using the plugin. 
The reasoning was, that they wanted to see the configuration, so instead of clicking twice, Once to go to the template job, and then press configure to see the actual configuration. They just had to click once to see the configuration.